tests/Feature/ValidationLogTest.php

```diff
@@ -28,6 +28,8 @@ class ValidationLogTest extends TestCase
     protected $custodian;
     protected $project;
 
+    protected bool $shouldFakeQueue = false;
```

Comment:
This is dangerous if we ever choose to bring prod data back to other envs for debugging. Just FYI.

Reply:
I see you've added to TestClass further down, so may need to understand why this one is allowed through